src/Kernel.php

```diff
@@ -66,5 +68,22 @@ protected function prepareCacheDir(string $staticCacheDir, string $tempCacheDir)
             // and copy all other files, i had intermediate problems when linking them
             $filesystem->copy("$staticCacheDir/$item", "$tempCacheDir/$item");
         }
+
+        $this->logToStderr(sprintf(
+            'cache directory prepared in %s ms.',
+            number_format((microtime(true) - $startTime) * 1000, 2)
+        ));
+    }
+
+    /**
+     * This method logs to stderr.
+     *
+     * It must only be used in a lambda environment since all error output will be logged.
+     *
+     * @param string $message The message to log
+     */
+    protected function logToStderr(string $message): void
```

Comment:
Why not private?

Reply:
Good question.

I assume you might want to overwrite the copy method (which is protected too) or some of the other methods and log too since there is no logging available in the kernel.
Or you might want to overwrite the log method to not log anything or use an existing logger that you somehow have available here before the container.

I also thought about using monolog or some other psr-3 logger but i didn't want to add a dependency. But then i'd make it private because you can just create your own logger the same way.

Reply:
Okey, fair. 

There is a bref/logger package. But it makes little sense to use it with Symfony as the symfony/http-kernel included a PSR-3 logger. But I think PSR-3 is overkill here.

Reply:
Okey, fair. 

There is a bref/logger package. But it makes little sense to use it with Symfony as the symfony/http-kernel included a PSR-3 logger. But I think PSR-3 is overkill here.

Reply:
Ohh interesting, `Symfony\Component\HttpKernel\Log\Logger` is included within the http-kernel component which we have a hard dependency on anyways so I can use it. And it is designed to log to `stderr` so it's basically perfect.

I can just create an instance of it within the constructor and put it into a private property instead of implementing my own with formatting etc.
And i don't think that is overkill since I want to log a lot more in #9 so it might not be a bad idea to use something that also has log levels.

Or what do you think?

Reply:
I dont see the within this PR. 

Lets start to use it when we implement #9 and we can evaluate it again then,